test/optim/test_optim.py

```diff
@@ -1805,8 +1754,7 @@ def test_rprop(self):
                     [param], lr=0.001, maximize=True, foreach=foreach
                 )
             )
-            with self.assertRaisesRegex(ValueError, "Invalid eta values: 1.0, 0.5"):
-                Rprop(None, lr=1e-2, etas=(1.0, 0.5), foreach=foreach)
```

Comment:
And this is the other place where coverage "decreased", since I no longer test this error case for BOTH foreach T and F. Which I believe is unnecessary anyway.